modules/cats-effect/build.sbt

```
@@ -3,7 +3,7 @@ import Utilities._
 
 name := "pureconfig-cats-effect"
 
-crossScalaVersions := Seq(scala212, scala213)
+crossScalaVersions := Seq(scala212, scala213, scala30)
```

We're currently following Scala team's recommendation (see https://scala-lang.org/blog/2021/10/21/scala-3.1.0-released.html#compatibility-notice) of publishing libraries with Scala 3.0 while still attempting to build them using the latest Scala 3.x version. See https://github.com/pureconfig/pureconfig/pull/1203 for more context. 

As such, we should also be compiling this for Scala 3.1.

```suggestion
crossScalaVersions := Seq(scala212, scala213, scala30, scala31)
```